src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const mirrorRobot = {};
```

Comment:
I think it is not necessary to create a variable. 

You can change the object (robot) because if you use this object in the function it is another object which exists only in this function)))

But it is not a mistake)) Good job :D

Reply:
Manipulations with robot parameter inside the function will mutate the initial object. And we have a request to return a new object in our task.

Reply:
@roshirin is totally right